clar2wasm/tests/standard/property_tests.rs

```diff
@@ -185,3 +193,179 @@ fn prop_bit_shift_right_int() {
         a.wrapping_shr((b % 128) as u32)
     })
 }
+
+#[test]
+fn prop_0_pow_uint_something_is_zero() {
+    let (instance, store) = load_stdlib().unwrap();
+    let store = RefCell::new(store);
+    let fun = instance
+        .get_func(store.borrow_mut().deref_mut(), "pow-uint")
+        .unwrap();
+
+    for st_a in UNSIGNED_STRATEGIES {
+        proptest! {|(m in st_a())| {
+                let mut res = [Val::I64(0), Val::I64(0)];
+                let res_slice = u128::relevant_slice(&mut res);
+
+                fun.call(
+                    store.borrow_mut().deref_mut(),
+                    &[Val::I64(0), Val::I64(0), m.low().into(), m.high().into()],
+                    res_slice,
+                )
+                .unwrap_or_else(|_| panic!("Could not call exported function pow-uint"));
+                let wasm_result = u128::from_wasm_result(res_slice);
+
+                prop_assert_eq!(if u128::from(m) == 0 {1} else {0}, wasm_result);
+            }
+        };
+    }
+}
+
+#[test]
+fn prop_somethin_pow_uint_zero_is_one() {
```

Comment:
"somethin" -> "something"

Reply:
fixed in 0c9b7deea238a278e9dae2abfc9b66218467dcd5